frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/activity-card/activity-card.tsx

```diff
@@ -58,27 +58,27 @@ const OngoingActivity = withDashboardResources(
         stopWatchPrometheusQuery(DATA_RESILIENCY_QUERY[StorageDashboardQuery.RESILIENCY_PROGRESS]);
     }, [watchPrometheus, stopWatchPrometheusQuery]);
 
-    const resiliencyProgress = prometheusResults.getIn([
+    const progressResponse = prometheusResults.getIn([
       DATA_RESILIENCY_QUERY[StorageDashboardQuery.RESILIENCY_PROGRESS],
       'data',
     ]) as PrometheusResponse;
-    const resiliencyProgressError = prometheusResults.getIn([
+    const progressError = prometheusResults.getIn([
       DATA_RESILIENCY_QUERY[StorageDashboardQuery.RESILIENCY_PROGRESS],
       'loadError',
     ]);
     const prometheusActivities = [];
     const resourceActivities = [];
 
-    if (isDataResiliencyActivity(resiliencyProgress)) {
+    if (getResiliencyProgress(progressResponse) < 1) {
       prometheusActivities.push({
-        results: resiliencyProgress,
-        loader: () => import('./data-resiliency-activity').then((m) => m.DataResiliencyActivity),
+        results: progressResponse,
+        loader: () => import('./data-resiliency-activity').then((m) => m.DataResiliency),
       });
     }
 
     return (
       <OngoingActivityBody
-        loaded={resiliencyProgress || resiliencyProgressError}
+        loaded={progressResponse || progressError}
```

Comment:
nit
```suggestion
        loaded={!!progressResponse || !!progressError}
```

Reply:
Implicit conversion is happening.
Its fine, can ignore IMO

Reply:
Readability. Similar issues with types in `consts`, although we know the return type from the function we still choose to put the type on it for the sake of readability. And of course precedents: https://github.com/openshift/console/blob/master/frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/inventory-card.tsx#L86

Reply:
Its in all activity cards now, hence I prefer ignoring.
Would be fine to send later on with changes everywhere.

Reply:
nope can't. Have other priorities.